src/sentry/api/endpoints/organization_events.py

```diff
@@ -132,6 +132,10 @@ def get(self, request, organization):
             )
             if not has_global_views and len(params.get("project_id", [])) > 1:
                 raise ParseError(detail="You cannot view events from multiple projects.")
+            if len(request.GET.getlist("field")) > 20:
```

Comment:
Should we have a constant for this on the server side as well? Have we checked to see how many columns customers are using in production? Will this new limit break any existing queries they have saved?

Reply:
What's a good place to check this in production?

Reply:
Redash is where you want to look.

Reply:
There are a few saved queries that have over 20 fields, and I checked with @wmak that none of them actually work right now anyways. So they'll get an error to reduce the number of columns instead now.